pkgs/build-support/vm/default.nix

```diff
@@ -982,10 +732,6 @@ rec {
       inherit expr;
     };
 
-  # The set of supported RPM-based distributions.
-
-  rpmDistros = { };
-
```

Comment:
FWIW, I deliberately left this stuff around when removing all the existing EOL versions in case anyone wanted to re‐add support for newer ones. But it’ll survive in the commit log anyway, so meh.